--- qwen3_5.py	2026-05-03 16:56:06.591029389 +0200
+++ qwen3_5.py	2026-05-03 16:59:18.794845508 +0200
@@ -283,8 +283,6 @@
             # mlp
             ("gate_up_proj", "gate_proj", 0),
             ("gate_up_proj", "up_proj", 1),
-            ("in_proj_ba", "in_proj_b", 0),
-            ("in_proj_ba", "in_proj_a", 1),
         ]
 
         if self.enable_lora:
@@ -461,7 +459,7 @@
         "gate_up_proj": ["gate_proj", "up_proj"],
         # GDN fused projections.
         "in_proj_qkvz": ["in_proj_qkv", "in_proj_z"],
-        "in_proj_ba": ["in_proj_b", "in_proj_a"],
+        "in_proj_ba": ["in_proj_ba.0", "in_proj_ba.1"],
     }
 
     def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
@@ -580,7 +578,7 @@
 
     packed_modules_mapping = Qwen3VLForConditionalGeneration.packed_modules_mapping | {
         "in_proj_qkvz": ["in_proj_qkv", "in_proj_z"],
-        "in_proj_ba": ["in_proj_b", "in_proj_a"],
+        "in_proj_ba": ["in_proj_ba.0", "in_proj_ba.1"],
     }
 
     def __init__(self, *, vllm_config: VllmConfig, prefix: str = "model"):
